src/credentials/sso/clients.ts

```diff
@@ -161,6 +162,11 @@ export class SsoClient {
         }
     }
 
+    public async logout(request: Omit<LogoutRequest, OmittedProps> = {}) {
+        const method = this.client.logout.bind(this.client)
+        await this.call(method as ExtractOverload<typeof method>, request)
```

Comment:
this bind() + ExtractOverload pattern is hard to follow

Reply:
It's kind of a TS hack specific to JS SDK v3. Can probably make it nicer.